sdk-http-apache-async/src/test/java/io/sphere/sdk/http/Tls12Test.java

```diff
@@ -0,0 +1,51 @@
+package io.sphere.sdk.http;
+
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.conn.ssl.DefaultHostnameVerifier;
+import org.apache.http.conn.ssl.SSLConnectionSocketFactory;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.ssl.SSLContexts;
+import org.junit.Test;
+
+import javax.net.ssl.SSLContext;
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+
+public class Tls12Test {
```

Comment:
May be we should make this part of another PR?

Reply:
Yeah, this should be part of a diff PR, would remove it 👍